cfme/configure/access_control/__init__.py

```diff
@@ -133,6 +135,7 @@ class User(Updateable, Pretty, BaseEntity):
     credential = attr.ib(default=None)
     email = attr.ib(default=None)
     group = attr.ib(default=None)
+    groups = attr.ib(default=None)
```

Comment:
**REQUIRED:** I really think we should change ```group``` to be ```groups```. Yes we will need to change it in more places, but to me it makes much more sense to pass in either a single item and for it to be transformed into a 1 len list, rather than have two params, that's just confusing.

Reply:
Updated so that UserCollection.create accepts a single 'groups' args that will accept a list of groups --OR-- a single group obj for backwards compatibility